### DISCUSSION TITLE
official stage editor? just askin' lol

### DISCUSSION THREAD

**Discussion by boyfriendfnflel** [FunkinCrew/Funkin#3595] [Ideas]
i wonder if y'all will make an official stage editor? like, having the stage editor be underneath character editor, and, well, if it DOES become real, it'd have to be in a new file format called .FNFS file (kinda like how the chart editor has the new chart file called .FNFC)

**Comment by AbnormalPoof** [comment]
There's already a stage editor, it's just really scuffed, #3482 plans to rework it.

**Reply by boyfriendfnflel** [reply]
w a i t -

**Reply by boyfriendfnflel** [reply]
i was probably ignorant with the stage editor itself :/

**Reply by boyfriendfnflel** [reply]
apologies
